### PR TITLE
Fix utils.get_env to return nil instead of empty string

### PR DESCRIPTION
**Pull request by dorkster100**

### CODE REVIEW

File: lua/laravel/application/init.lua [hunk 1]
```diff
@@ -16,15 +16,16 @@ local initialize = function(options)
...
   -- fill with the environment
-  if not env then
+  if not resolvedEnv then
     utils.notify("App", { msg = "Could not initialize environment", level = "ERROR" })
     return
   end
 
   app = {
-    environment = env,
+    envSetup = resolvedEnv,
```
Comment (adalessa) [inline lua/laravel/application/init.lua:27]: what will be the advantage of doing this ?

File: lua/laravel/environment/docker_compose.lua [hunk 2]
```diff
@@ -3,30 +3,32 @@ local utils = require "laravel.utils"
 local M = {}
 
 ---@param opts table|nil
----@return table
+---@return function
 M.setup = function(opts)
```
Comment (adalessa) [inline lua/laravel/environment/docker_compose.lua:7]: I don't like this of returning another function

File: lua/laravel/application/init.lua [hunk 3]
```diff
@@ -63,6 +64,7 @@ local build_command = function(command, args)
 end
 
 local warmup = function()
+  app.environment = app.envSetup()
```
Comment (adalessa) [inline lua/laravel/application/init.lua:67]: Set up is not warm up.

File: lua/laravel/utils.lua [hunk 4]
```diff
@@ -38,11 +38,18 @@ utils.get_vsel = function()
...
   else
-    return vim.fn.eval("$" .. var)
+    envVal = vim.fn.eval("$" .. var)
   end
+
+  if envVal == "" then
+    return nil
+  end
+
+  return envVal
```
Comment (adalessa) [inline lua/laravel/utils.lua:52]: This looks good, and matches the idea on the name of the PR, the rest is confusing.

**Comment by dorkster100** [comment]
i will redo PR, it is a lot smaller

**Pull request closed without merging**